src/main/java/igym/entities/UserEntity.java

```diff
@@ -6,31 +6,39 @@
 
 @Table(name = "users")
 @Entity
-public class User {
-    
+public class UserEntity {
```

Comment:
```suggestion
public class User {
```

There are other ways to provide more context in naming. You can use UserGym, UserTrainer... Names related to the context of application.
Following the principles of clean code you should avoid unnecessary words or noisy suffixes such as Entity, Object, or Data. 
"Names should not be redundant and should reveal their purpose. Use descriptive names but avoid unnecessary qualifiers."

Reply:
those are good suggestions, I will keep it as User,